src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/domain/ccd/message/EventInformation.java

```diff
@@ -17,7 +17,7 @@
 @ToString
 @Builder
 @EqualsAndHashCode
-@SuppressWarnings("PMD.ExcessiveParameterList")
+@SuppressWarnings({"PMD.ExcessiveParameterList", "PMD.UnnecessaryFullyQualifiedName"})
```

Comment:
Don't think we need this suppression?

Reply:
No, i've moved the method causing the warning to the right class